hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/DefaultServiceEntityEnricherTest.java

```diff
@@ -214,15 +211,15 @@ public void test_case5_findServiceNameOfFirstAncestorThatIsNotAnExitSpanAndBelon
   public void test_case6_findServiceNameOfFirstAncestorThatIsNotAnExitSpanAndBelongsToADifferentService() {
     // Case 6:  sampleapp (entry) -> sampleapp (entry) -> sampleapp (exit)
     Event parent1 = createEvent("parent1",
-        Map.of("span.kind", "server", JAEGER_SERVICE_NAME_ATTR_NAME, "sampleapp"),
-        Map.of(API_BOUNDARY_TYPE_ATTR, "ENTRY", "SERVICE_NAME", "sampleapp"), null);
+        Map.of("span.kind", "server"),
+        Map.of(API_BOUNDARY_TYPE_ATTR, "ENTRY", "SERVICE_NAME", "sampleapp"), null, "sampleapp");
```

Comment:
Just curious - where is this `"SERVICE_NAME", "sampleapp"` used?

Reply:
That shouldn't be need with this change.

Reply:
@buchi-busireddy it's needed, as method ```findServiceNameOfFirstAncestorThatIsNotAnExitSpanAndBelongsToADifferentService``` gets parent serviceName from enriched serviceName